backend/utils/mail.py

```diff
@@ -3,17 +3,28 @@
 from email.mime.text import MIMEText
 import os
 from dotenv import load_dotenv
+from sqlalchemy.orm import Session
+from sqlalchemy import create_engine
+from db.orm import Setting
+from typing import Optional
 
-load_dotenv()
 
-SMTP_SERVER = os.getenv("SMTP_SERVER", "smtp.gmail.com")
-SMTP_PORT = int(os.getenv("SMTP_PORT", 587))
-EMAIL_SENDER = os.getenv("EMAIL_SENDER")
-EMAIL_PASSWORD = os.getenv("EMAIL_PASSWORD")
+load_dotenv()
+DATABASE_URL = os.getenv("DATABASE_URL")
+engine = create_engine(DATABASE_URL)
 
+def get_setting(key: str) -> Optional[str]:
```

Comment:
Tuto by malo byť minimálne cache, a dá sa to spraviť že v jednom dotaze vyberieš všetky údaje naraz, nie sekvenčne dotaz po dotaze (to je strašne pomalé)

Teraz pri každom maile berieme konfig z databázy, a to chcem minimalizovať (mohla by tu byť globálna premenná s konfigom, čiže pri štarte aplikácie sa konfig uloží do premennej a funkcia s tým bude pracovať, pozri aj functools lre_cache v Python)